src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -326,6 +326,9 @@ private static Process StartNewVisualStudioProcess(string installationPath, int
                 // Disable roaming settings to avoid interference from the online user profile
                 Process.Start(vsRegEditExeFile, $"set \"{installationPath}\" {Settings.Default.VsRootSuffix} HKCU \"ApplicationPrivateSettings\\Microsoft\\VisualStudio\" RoamingEnabled string \"1*System.Boolean*False\"").WaitForExit();
 
+                // Disable async completion for 16.0 Preview 3 testing
```

Comment:
Do we have abug tracking turning this back on? Should reference it here

Reply:
Enabling async completion in a public stable release is dependent on at least one successful integration test pass completing with it enabled, so we're conceptually tracking it at a high level.

Reply:
➡️ Filed #33797